env_vars

```diff
@@ -0,0 +1,14 @@
+readonly HOME_DIR="$(pwd)"
+readonly TMP_DIR="${HOME_DIR}/tmp"
+
+readonly FILES_BASE="http://files.squeak.org/base"
+readonly RELEASE_URL="${FILES_BASE}/${SMALLTALK_VERSION/Etoys/Squeak}"
```

Comment:
Why is there an `Etoys`?

Reply:
There is still support for building Etoys bundles. See http://files.squeak.org/etoys/ . It uses the same base files from files.squeak.org/base. So, you have to replace "Etoys" with "Squeak" when building for "Etoys-trunk" or "Etoys64-trunk".